Week7/app/src/main/java/com/kulloveth/covid19virustracker/data/Injection.kt

```diff
@@ -0,0 +1,24 @@
+package com.kulloveth.covid19virustracker.data
+
+import androidx.lifecycle.ViewModelProvider
+import com.kulloveth.covid19virustracker.api.buildApiService
+import com.kulloveth.covid19virustracker.data.db.CovidDatabase
+import com.kulloveth.covid19virustracker.ui.ViewModelFactory
+
+object Injection {
+    // build status api
+     val apiService by lazy { buildApiService("https://corona-virus-stats.herokuapp.com/api/") }
+    //build news api
+    val newsApiService by lazy { buildApiService("https://newsapi.org/") }
+
+     val db = CovidDatabase.invoke()
```

Comment:
I like how you set up this repository pattern and nice job with the injection too.